### ISSUE TITLE
Finding localized to the penis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10125]
**Preferred term label:**
Finding localized to the penis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mondor disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000052
- label: Localized to the penis
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to a finding that is situated or occurring in or on the penis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10125
- synonyms:
